exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetMetadataCache.java

```diff
@@ -495,14 +495,14 @@ public void testFutureUnsupportedMetadataVersion() throws Exception {
     try {
       test("use dfs_test.tmp");
       test("create table `%s` as select * from cp.`tpch/nation.parquet`", unsupportedMetadataVersion);
-      String lastVersion = Iterables.getLast(MetadataVersion.Constants.SUPPORTED_VERSIONS);
-      for (String supportedVersion : MetadataVersion.Constants.SUPPORTED_VERSIONS) {
-        if (new MetadataVersion(lastVersion).compareTo(new MetadataVersion(supportedVersion)) < 0) {
+      MetadataVersion lastVersion = Iterables.getLast(MetadataVersion.Constants.SUPPORTED_VERSIONS);
+      for (MetadataVersion supportedVersion : MetadataVersion.Constants.SUPPORTED_VERSIONS) {
+        if (lastVersion.compareTo(supportedVersion) < 0) {
           lastVersion = supportedVersion;
         }
       }
       // Get the future version, which is absent in MetadataVersion.SUPPORTED_VERSIONS list
-      String futureVersion = "v" + (Integer.parseInt(String.valueOf(lastVersion.charAt(1))) + 1);
+      String futureVersion = "v" + (Integer.parseInt(String.valueOf(lastVersion.toString().charAt(1))) + 1);
```

Comment:
`String futureVersion = new MedataVersion(lastVersion.getMajor() + 1, 0).toString();`
Just add getters for major and minor versions to `MedataVersion` class first.